src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -589,6 +590,12 @@ private DataAccessRequest updateDarWithDocumentContents(
       DataAccessRequest dar,
       InputStream uploadInputStream,
       FormDataContentDisposition fileDetail) throws IOException {
+    //when we move updateDarWithDocumentContents to the service tier, we should incorporate the code
+    //below into that method
```

Comment:
The DUOS code isn't consistent, but I think we should move toward the google java code style. Following that guide, `//` comments must have at least once space before its text.
```suggestion
    // When we move updateDarWithDocumentContents to the service tier, we should incorporate the code
    // below into that method.
```